src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+      
+        <div class="question">
+          <label for="surname">Surname:</label> 
+
+          <input 
+            type="text" 
+            autocomplete="off"
+            name="surname" 
+            id="surname"
+            minlength="2"
+            maxlength="30"
+          >
+        </div>
+
+        <div class="question">
+          <label for="name">Name:</label> 
+
+          <input 
+            type="text" 
+            autocomplete="off"
+            name="name" 
+            id="fullName"
+            minlength="2"
+            maxlength="30"
+          >
+        </div>
+        
+        <div class="question">
+          <label for="age">How old are You?</label> 
+
+          <input 
+            type="number" 
+            name="age" 
+            min="1" 
+            max="100" 
+            value="12" 
+            required
+            id="age"
+          >
+        </div>
+        
+        <div class="question">
+          <label for="dob">Full date of birth:</label> 
+
+          <input 
+            type="date" 
+            name="dob"
+            id="dob"
+          >
+        </div>
+        
+        <div>
+          <label for="consent">I accept the term of the agreement</label> 
+
+          <input 
+            type="checkbox" 
+            name="consent"
+            id="consent"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="question">
+          <label for="@address">E-mail:</label> 
+
+          <input 
+            type="email" 
+            name="@address" 
+            placeholder="email@example.com"
+            id="@address"
+          >
+        </div>
+
+        <div>
+          <label for="password">Password:</label> 
+
+          <input 
+            type="password" 
+            name="password" 
+            required
+            id="password"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="question">
+          <label for="cats">Do you love cats?</label> 
+
+          <input 
+            type="radio" 
+            name="cats"
+            id="yes"
+          > Yes
+          <input 
+            type="radio" 
+            name="cats"
+            id="cats"
+          > No
```

Comment:
do not forget about code formatting, please
```suggestion
          <input 
            type="radio" 
            name="cats"
            id="yes"
          >
          Yes
          <input 
            type="radio" 
            name="cats"
            id="cats"
          >
          No
```